docs/az/docs/index.md

```diff
@@ -0,0 +1,469 @@
+
+{!../../../docs/missing-translation.md!}
+
+
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI framework, yüksək performans, sadə öyrənmək, çevik kodlamaq, istehsal üçün hazır</em>
+</p>
+<p align="center">
+<a href="https://github.com/tiangolo/fastapi/actions?query=workflow%3ATest" target="_blank">
+    <img src="https://github.com/tiangolo/fastapi/workflows/Test/badge.svg" alt="Test">
+</a>
+<a href="https://codecov.io/gh/tiangolo/fastapi" target="_blank">
+    <img src="https://img.shields.io/codecov/c/github/tiangolo/fastapi?color=%2334D058" alt="Coverage">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/v/fastapi?color=%2334D058&label=pypi%20package" alt="Package version">
+</a>
+</p>
+
+---
+
+**Sənədlər**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**Mənbə kodu**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI müasir, sürətli (yüksək performanslı), standart Python tipli göstərişlərə əsaslanaraq Python 3.6+ ilə API qurmaq üçün veb framework-ü.
+
+Aşağıdakılar əsas xüsusiyyətlərdir:
+
+* **Sürətli**: Çox yüksək performans, **NodeJS** və **Go** ilə bərabər (Starlette və Pydantic-ə təşəkkürlər). [Ən sürətli Python frameworklərindən biri](#performans).
```

Comment:
"ilə bərabər" would mean it is together with these frameworks. Replacing it with "səviyyəsində" will keep the same meaning as in English.